src/code/fv-components/fv-gene-label.js

```diff
@@ -13,6 +13,10 @@ const Form = React.createClass({
     // render :: a -> ReactElement
     render: function(){
         return <SimpleSelect defaultValue={this.props.defaultOption} onValueChange={this.props.handleChange} options = {this.props.options} hideResetButton={true} editable={false} disabled={false}></SimpleSelect>;
+    },
+
+    componentDidMount: function() {
+      document.querySelectorAll("input").forEach(input => input.disabled = true);
```

Comment:
This seems to have the side-effect that it won't notice you clicking away from the pulldowns to dismiss them, because they are no longer focused and so no longer blur.

I tested this out instead with 

```
document.querySelectorAll("input").forEach(input.setAttribute('readonly', true));
```

and it seemed to work well on my Android phone, as far as I could see, but you would need to do more testing of it.